src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -388,6 +341,7 @@ typedef struct {
     /* Window/RMA Globals */
     void *win_map;
     uint64_t rma_issued_cntr;
+    void *mr_key_allocator;
```

Comment:
In #3493 I commented below:
>Also, if I understood correctly, this single allocator is now used for
> * generating window ID for RMA (=equal to MR key for RMA)
> * generating MR key for pt2pt LMT
> These design choices have to be written somewhere, probably here.

I think we need more documentation like this as a comment as well as commit message. @yfguo @alexander-sannikov

Reply:
Commit: "per-comm MR-key allocator is replaced with global one"

The commit message totally lacks *why* we had to do replacement.